common/tools/dev-tool/src/config/rollup.base.config.ts

```diff
@@ -103,11 +111,15 @@ function makeOnWarnForTesting(): (warning: RollupWarning, warn: WarningHandler)
 
 // #endregion
 
-export function makeBrowserTestConfig(): RollupOptions {
+export function makeBrowserTestConfig(pkg: PackageJson): RollupOptions {
```

Comment:
This function is exported, could you update all calls to it? there is one call in [schema registry](https://github.com/Azure/azure-sdk-for-js/blob/5ec971e7538ad75a1d9e3701a673d82af8faf79d/sdk/schemaregistry/schema-registry-avro/rollup.patched.config.js#L15).

Reply:
Oops! good call, thanks 😄

Reply:
I updated it at the call because that's the only one I found and seemed to be a one-off case - but I was debating whether to make `pkg` an optional param and default to `dist-esm/src/index.js` if it's not provided like in https://github.com/Azure/azure-sdk-for-js/blob/main/common/tools/dev-tool/src/config/rollup.base.config.ts#L162

What do you all think?

Reply:
> edit: looking at it again, I guess if base config uses a default value if pkg["module"] does not exist maybe I should too...

Sounds good to me! Thanks!